fabfile.py

```diff
@@ -92,7 +92,12 @@ def decrypt_secrets(c):
 def install_deps(c: Connection):
     print('-- Installing dependencies')
     with c.cd(c.release_path):
-        c.run('source .venv/bin/activate && pipenv install --deploy', echo=True, env={'PIPENV_VENV_IN_PROJECT': 'true'})
+        # TODO: Edit out pipenv dependency
+        c.run("python3 -m venv .venv")
+        c.run("source .venv/bin/activate")
+        c.run(".venv/bin/python -m pip install --upgrade pip")
+        c.run(".venv/bin/python -m pip install --upgrade setuptools")
+        c.run(".venv/bin/python -m pip install -r requirements.txt")
```

Comment:
I think deduplicating this with the Makefile would be good, but probably should not install the dev packages. That's a bit more difficult to do as-is and would probably need a new Makefile target that only installs the prod packages.

Reply:
Separated out the dev and prod dependencies into two separate commands, refactored fabfile to use the prod version